packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1128,25 +1128,26 @@ The performance optimization will only be applied when all three parameters are
 
 ##### Request parameters
 
-The first three parameters in the table below correspond to different types of lists:
+The first three parameters in the table below correspond to different ways to list users.
 
-- [List All Users](#list-all-users) (no parameters)
-- [Find Users](#find-users) (`q`)
-- [List Users with a Filter](#list-users-with-a-filter) (`filter`)
 - [List Users with Search](#list-users-with-search) (`search`)
+- [List Users with a Filter](#list-users-with-a-filter) (`filter`)
+- [Find Users](#find-users) (`q`)
+- [List All Users](#list-all-users) (no parameters)
 
 | Parameter   | Description                                                                                                                                    | Param Type   | DataType   | Required |
 | :---------- | :--------------------------------------------------------------------------------------------------------------------------------------------- | :----------- | :--------- | :------- |
-| q           | Finds a user that matches `firstName`, `lastName`, and `email` properties                                                                      | Query        | String     | FALSE    |
+| search      | Searches for users with a supported [filtering](/docs/reference/core-okta-api/#filter) expression for most properties. Prefer this option for optimal performance         | Query        | String     | FALSE    |
 | filter      | [Filters](/docs/reference/core-okta-api/#filter) users with a supported expression for a subset of properties                  | Query        | String     | FALSE    |
-| search      | Searches for users with a supported [filtering](/docs/reference/core-okta-api/#filter) expression for most properties          | Query        | String     | FALSE    |
-| limit       | Specifies the number of results returned (maximum 200)                                                                                         | Query        | Number     | FALSE    |
+| q           | Finds a user that matches `firstName`, `lastName`, and `email` properties (but prefer search; see above)                       | Query        | String     | FALSE    |
+| limit       | Specifies the number of results returned (maximum 200). Okta recommends preferring search; see above                           | Query        | Number     | FALSE    |
```

Comment:
`Okta recommends preferring search; see above` seems to have been added at a wrong place?

Reply:
Yeah, unless you're saying "don't try limiting to fewer results, switch to search"?

Reply:
🤦 
Quite right.  Fixed.